Makefile

```diff
@@ -33,3 +33,10 @@ wsgi-two-apps: templates
 
 bottle: templates
 	python apps/bottle_app.py $(HOST) $(PORT)
+
+# why does this not work?
```

Comment:
i really don't know why it doesn't work to turn on the server, but I'm fine leaving it as is and when we do actually add fastapi support to figure it out